assets/css/default.css

```diff
@@ -5,6 +5,17 @@ body {
     Arial, sans-serif;
 }
 
+#player {
```

Comment:
Sorry I didn't mention this correctly: this is only needed for the embed page. It should either be in `embed.ecr` or a separate CSS file.